src/ctl.c

```diff
@@ -3083,3 +3085,44 @@ experimental_hooks_remove_ctl(tsd_t *tsd, const size_t *mib, size_t miblen,
 label_return:
 	return ret;
 }
+
+static int
+experimental_frag_ctl(tsd_t *tsd, const size_t *mib, size_t miblen,
+    void *oldp, size_t *oldlenp, void *newp, size_t newlen) {
+	assert(oldp != NULL && newp != NULL);
+
+	/* trim *oldlenp and newlen so that:
```

Comment:
Check the coding style here: https://github.com/jemalloc/jemalloc/wiki/Coding-Style